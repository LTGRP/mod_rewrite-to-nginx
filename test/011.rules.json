[
    {
        "type": "status",
        "code": 403,
        "match": "\\.exe"
    }
]
