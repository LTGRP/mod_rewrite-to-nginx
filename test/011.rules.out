if ($request_uri ~* \.exe) {
    return 403;
}
